Tools/autotest/common.py

```diff
@@ -196,7 +196,10 @@ def fetch_parameters(self):
     def reboot_sitl(self):
         """Reboot SITL instance and wait it to reconnect."""
         self.mavproxy.send("reboot\n")
-        self.mavproxy.expect("tilt alignment complete")
+        self.mav.wait_heartbeat() # makes sure we have a mav_type
```

Comment:
I don't think this is autotest proof ... we could still get a heartbeat just after issuing a reboot to mavproxy.